samples/backchannel-inject-data-on-post/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <title>Web Chat: Inject data on post activity</title>
+    <!--
+      For demonstration purpose, we are using development branch of Web Chat at "/master/webchat.js".
```

Comment:
Please copy from latest full-bundle sample -- fixed verbiage

Reply:
Hi , I tried but didn't find the sample for 'Inject data on post activity', Could I have a link here? Thank you!

Reply:
Hi, if i send this custom data to bot, how can i retrieve the message from bot side? Thank you.